mapping.py

```diff
@@ -0,0 +1,19 @@
+import collections
+from operator import index
```

Comment:
Are you using this (`from operator import index`) anywhere?  If not, we should delete it.

Reply:
removed